packages/logger/tests/helpers/InvocationLog.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Log level. used for filtering the log
+ */
+export enum LEVEL {
+  DEBUG = 'DEBUG',
+  INFO = 'INFO',
+  WARN = 'WARN',
+  ERROR = 'ERROR',
+}
+
+export type FunctionLog = {
+  timestamp: string
+  invocationId: string
+  logLevel: LEVEL
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  logObject: {[key: string]: any}
+};
+export class InvocationLogs {
+  public static LEVEL = LEVEL;
+
+  /**
+   * Array of logs from invocation. 
+   * 
+   * The first element is START, and the last two elements are END, and REPORT.
+   * In each log, each content is separated by '\t'
+   *    [
+   *      'START RequestId: afb6de1a-48f8-4fbb-ad72-23a4f0c2924c Version: $LATEST',
+   *      '2022-01-27T16:04:39.323Z\tafb6de1a-48f8-4fbb-ad72-23a4f0c2924c\tINFO\t{"cold_start":true,"function_arn":"arn:aws:lambda:eu-central-1:561912387782:function:loggerMiddyStandardFeatures-c555a2ec-1121-4586-9c04-185ab36ea34c","function_memory_size":128,"function_name":"loggerMiddyStandardFeatures-c555a2ec-1121-4586-9c04-185ab36ea34c","function_request_id":"7f586697-238a-4c3b-9250-a5f057c1119c","level":"INFO","message":"This is an INFO log with some context and persistent key","service":"logger-e2e-testing","timestamp":"2022-01-27T16:04:39.323Z","persistentKey":"works"}',
```

Comment:
Can we use the same ARNs we have [in the docs](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/logger/#capturing-lambda-context-info)?